pipeline-modules/app-service/aws/make-target.yaml

```diff
@@ -53,6 +54,6 @@ template: |
     build:
       commands:
         {% for target in targets %}
-        - cd {{ workspace }}/{{ working_dir }}/{{ target.target_dir }}
+        - cd $CODEBUILD_SRC_DIR/{{ workspace }}/{{ working_dir }}/{{ target.target_dir }}
```

Comment:
workspace is already set to $CODEBUILD_SRC_DIR

Reply:
oh right, fixing.